tests/conftest.py

```diff
@@ -69,7 +69,7 @@ def dbt_profile_target(profile_type, bv_server_process, tmp_path_factory):
                 "Please set the MOTHERDUCK_TOKEN environment variable to run tests against MotherDuck"
             )
         profile["disable_transactions"] = True
-        profile["path"] = "md:test"
+        profile["path"] = "md:plugin_test"
```

Comment:
this will break for main b/c the "md:test" stuff is the namespace I use on my own MD account for testing purposes, so let's take it out prior to merging